frontend/test/metabase/query_builder/components/NativeQueryEditor.cy.spec.js

```diff
@@ -0,0 +1,42 @@
+import { signInAsNormalUser } from "__support__/cypress";
+describe("NativeQueryEditor", () => {
+  beforeEach(() => {
+    signInAsNormalUser();
+    cy.on("uncaught:exception", (err, runnable) => {
+      // ignore the error if it's a "script error"
+      if (err.message.match(/^Script error./)) {
```

Comment:
Cypress fails tests on any uncaught error. I think that's a good move on their part, but unfortunately, we have a handful of "benign" uncaught errors.

I first tried to fix the error (which I believe is coming from Ace). When I couldn't find it, I decided to just ignore it. Instead of ignoring all errors, I match the beginning of the Cypress message that wraps the error. It's still a broad category of errors but somewhat more specific to what I was looking for.